test/integration/upvs_messages_api_test.rb

```diff
@@ -0,0 +1,913 @@
+require "test_helper"
+
+class ThreadsApiTest < ActionDispatch::IntegrationTest
+  setup do
+    @key_pair = OpenSSL::PKey::RSA.new File.read 'test/fixtures/tenant_test_cert.pem'
+    @tenant = tenants(:ssd)
+    @before_request_messages_count = Message.count
+  end
+
+  test 'can upload valid message' do
+    message_params = {
+      posp_id: 'App.GeneralAgenda',
+      posp_version: '1.9',
+      message_type: 'App.GeneralAgenda',
+      message_id: SecureRandom.uuid,
+      correlation_id: SecureRandom.uuid,
+      sender_uri: 'SSDMainURI',
+      recipient_uri: 'ico://sk/12345678',
+      title: 'Všeobecná agenda',
+      objects: [
+        {
+          name: 'Form.xml',
+          is_signed: false,
+          to_be_signed: true,
+          mimetype: 'application/x-eform-xml',
+          object_type: 'FORM',
+          content: Base64.encode64('<?xml version="1.0" encoding="utf-8"?>
+<GeneralAgenda xmlns="http://schemas.gov.sk/form/App.GeneralAgenda/1.9" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <subject>Všeobecný predmet</subject>
+  <text>Všeobecný text</text>
+</GeneralAgenda>')
+        }
+      ]
+    }
+
+    post '/api/messages', params: message_params.merge({ token: generate_api_token(sub: @tenant.id, key_pair: @key_pair) }).to_json, headers: { "Content-Type": "application/json;type=upvs" }
```

Comment:
Tento type by som posielal normalne v payloade toho message_params a mozno by som isiel tak daleko, ze by som to nazval type a dal tam Upvs::MessageDraft takto to rovno vieme prelozit na triedu a ta uz vie urobit load a validaciu.

Zaroven rozmyslam ci by to nemal byt endpoint /messages/drafts lebo to vytvara drafty.

Reply:
Premenovane na `api/messages/message_drafts`.